docs/tool-parsing.md

```
@@ -92,257 +113,54 @@ list_directory {"path": "/home/user"}
-
-### Examples
+### Gemma (TOOL_REQUEST)
 
-#### Basic Configuration
-
-```json
-{
-  "enableTextToolCallParsing": true,
-  "textToolCallModels": ["llama-3.2", "mixtral-instruct"]
-}
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)